commons/java/openjdk11/Dockerfile

```diff
@@ -2,3 +2,5 @@ FROM cloudsuitetest/debian:base-os
 
 RUN apt-get update -y && apt-get install -y --no-install-recommends openjdk-11-jdk-headless \
         && rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_HOME /usr/
```

Comment:
Is this the correct path? Shouldn't it be something like `/usr/lib/jvm/...` ?

Reply:
It works out of box though. I think `javac` is in /usr/bin/javac now

Reply:
You can check the CI result.

Reply:
This is not a good answer. You have to verify this manually by going inside the image. The check might pass due to a side effect (tests cannot prove the absence of bugs). Please double-check this. As far as I know, `JAVA_HOME` should be within that path that I wrote before.

Reply:
Checked. /usr/bin/javac is a link pointing to /etc/alternatives/javac, which is pointing to /usr/lib/jvm/java-11-openjdk-amd64/bin/javac.

Reply:
So as I said `JAVA_HOME` should be `/usr/lib/jvm/java-11-openjdk-amd64`. Please note `JAVA_HOME` is not only used to access `javac`.

Reply:
However, please note the path is different across architecture, and the above path is true only for the `amd64` image. So you need to set this properly for each image.

Reply:
Done.